spec/pluralrules.html

```diff
@@ -355,7 +351,6 @@ <h1>Properties of Intl.PluralRules Instances</h1>
       <li>[[MinimumIntegerDigits]] is a non-negative integer Number value indicating the minimum integer digits to be used.</li>
       <li>[[MinimumFractionDigits]] and [[MaximumFractionDigits]] are non-negative integer Number values indicating the minimum and maximum fraction digits to be used. Numbers will be rounded or padded with trailing zeroes if necessary.</li>
       <li>[[MinimumSignificantDigits]] and [[MaximumSignificantDigits]] are positive integer Number values indicating the minimum and maximum fraction digits to be used. Either none or both of these properties are present; if they are, they override minimum and maximum integer and fraction digits.</li>
-      <li>[[PluralCategories]] is an array of unique string values, from the the list `"zero"`, `"one"`, `"two"`, `"few"`, `"many"` and `"other"`, that are relevant for the locale whose localization is specified in LDML Language Plural Rules.</li>
```

Comment:
I think it'd be useful to keep the info about the possible plural category values and that the returned array doesn't contain duplicates somewhere in the spec.

Reply:
yes, let's keep this around somewhere! Maybe as a note in resolvedOptions.

Reply:
Good point. I've moved this text into resolvedOptions.